src/jit/lclvars.cpp

```diff
@@ -4242,7 +4286,23 @@ int Compiler::lvaAssignVirtualFrameOffsetToArg(unsigned lclNum, unsigned argSize
         }
 #endif // _TARGET_ARM_
 
-        varDsc->lvStkOffs = argOffs;
+#ifdef UNIX_AMD64_ABI
```

Comment:
It is not clear to me where the space for the spilled reg args is allocated and how you would skip the discontinuity in the arguments layout due to the ebp and the return address.